xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -435,31 +435,26 @@ CPVRRecordingPtr CPVRRecordings::GetById(int iClientId, const std::string &strRe
   return retVal;
 }
 
-CPVRRecordingPtr CPVRRecordings::GetByFileItem(const CFileItem &item) const
-{
-  const CPVRRecording *recording = item.GetPVRRecordingInfoTag();
-  return GetById(recording->m_iClientId, recording->m_strRecordingId);
-}
-
 void CPVRRecordings::Clear()
 {
   CSingleLock lock(m_critSection);
   m_recordings.clear();
 }
 
-void CPVRRecordings::UpdateFromClient(const CPVRRecording &tag)
+void CPVRRecordings::UpdateFromClient(const PVR_RECORDING &recording, unsigned int iClientId)
```

Comment:
this one needs to be changed back. c types belong in the add-on interface classes, not anywhere else in the code